cluster-tools/src/main/scala/org/apache/pekko/cluster/client/ClusterClient.scala

```diff
@@ -127,7 +127,7 @@ object ClusterClientSettings {
  *   external service registry
  */
 @deprecated(
-  "Use Akka gRPC instead, see https://doc.akka.io/docs/akka/2.6/cluster-client.html#migration-to-akka-grpc",
+  "Use Akka gRPC instead, see https://pekko.apache.org/docs/pekko/2.6/cluster-client.html#migration-to-akka-grpc",
```

Comment:
can you fix up the rest of this class - should not have `pekko/2.6` - use `pekko/current`